testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeHiveIcebergRedirections.java

```diff
@@ -43,7 +43,7 @@ public EnvSinglenodeHiveIcebergRedirections(DockerFiles dockerFiles, Standard st
     @Override
     public void extendEnvironment(Environment.Builder builder)
     {
-        builder.addConnector("hive", forHostPath(configDir.getPath("hive.properties")));
+        builder.addConnector("hive-hadoop2", forHostPath(configDir.getPath("hive.properties")));
```

Comment:
If we're using the legacy connector name in `hive.properties`, maybe it's better to change it there?